_opensearch/geohexgrid-agg.md

```diff
@@ -0,0 +1,377 @@
+---
+layout: default
+title: GeoHex grid aggregations
+parent: Aggregations
+nav_order: 4
+has_children: false
+---
+
+# GeoHex grid aggregations
+
+The Hexagonal hierarchical geospatial indexing system (H3) partitions the Earth's areas into identifiable hexagon-shaped cells. 
```

Comment:
This is a funny one. It's a proper name, so it should be written as Hexagonal Hierarchical Geospatial Indexing System (H3). Uber refers to it as H3 on first mention, ignoring capitalization standards for proper names. It's their product though, so they can do what they want. I'd follow capitalization standards for proper names in our documentation.